### PR TITLE
docs: versioned

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

Version docs, unfortunately because there is no dropdown in the theme the way one could go see docs related to v0.32 is through `/master`

I am not the biggest fan of this approach but unfortunately it all we have until https://github.com/cosmos/vuepress-theme-cosmos/issues/91 is completed

Closes: #XXX

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -11,126 +11,137 @@ module.exports = {
...
+            },
+            // TODO: remove once https://github.com/cosmos/vuepress-theme-cosmos/issues/91 is closed
+            {
+              title: 'Version 0.32',
+              path: '/v0.32',
+              static: true
+            },
+            {
+              title: 'Version 0.33',
+              path: '/v0.33',
```
Comment (cmwaters) [inline docs/.vuepress/config.js:45]: Are these valid paths?

**Comment by marbar3778** [comment]
/master:
Screen Shot 2020-08-20 at 8 49 54 PM

/0.33
Screen Shot 2020-08-20 at 8 49 45 PM

/0.32
Screen Shot 2020-08-20 at 8 49 38 PM


sidebar on this branch endpoint: 
Screen Shot 2020-08-20 at 8 51 52 PM

master will not render when it deploys.

**Comment by marbar3778** [comment]
@fadeev can you verify its correct. Also is this the correct order in version? Every time I run `npm run build` then view the generated code on a browser it starts off with /0.32 instead of master

**Comment by fadeev** [comment]
Merged into `master` locally (you have to merge into `master`, because `make build-docs` checks out master), builds fine. Both versions show up in the sidebar. `v0.32` shows up in an old design, but that's exactly what's expected. Looks good to me.

Screenshot 2020-08-21 at 00 36 12

File: Makefile [hunk 2]
```diff
@@ -170,6 +170,7 @@ build-docs:
 	cd docs && \
 	while read p; do \
 		(git checkout $${p} && npm install && VUEPRESS_BASE="/$${p}/" npm run build) ; \
+		git stash ; \
```
Comment (melekes) [inline Makefile:173]: Do we call `git stash pop` sometime later? if not, isn't `git checkout .` better?

Comment (marbar3778) [reply Makefile:173]: What does the `.` do? I haven't used this feature.

Comment (melekes) [reply Makefile:173]: It undoes unstaged local modifications

Comment (marbar3778) [reply Makefile]: thank you!!

**Review by erikgrinaker — APPROVED** [review]
LGTM!

**Pull request merged**